at_client/lib/src/manager/stream_manager.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:at_client/at_client.dart';
+import 'package:at_client/src/preference/at_client_preference.dart';
+import 'package:at_client/src/stream/at_stream_response.dart';
+
+class StreamManager {
+  late String _atSign;
+  static final Map<String, StreamManager> _syncManagerMap = {};
+
+  late AtClientPreference preference;
+
+  RemoteSecondary? remoteSecondary;
+
+  factory StreamManager.getInstance(String atSign) {
+    if (_syncManagerMap.containsKey(_syncManagerMap)) {
+      return _syncManagerMap[atSign]!;
+    }
+    final syncManager = StreamManager._internal(atSign);
+    _syncManagerMap[atSign] = syncManager;
+    return syncManager;
+  }
+
+  StreamManager._internal(this._atSign);
+
+  /// onDone[AtStreamResponse]
+  /// onError[AtStreamResponse]
+  Future<void> send(
+  String sharedWith, String filePath, Function onDone, Function onError) async {
+
+  }
+
+  Future<void> resume(
```

Comment:
Stream.sender.send.  Stream.recipient.resume

Reply:
In my opinion, we need the following:

Sender side:
-------------
Stream.Sender.send(), status(), cancel()

Recipient side:
---------------
Stream.Recipient.resume(), status(), cancel()